components/slack/sources/new-reaction-added/new-reaction-added.mjs

```diff
@@ -44,14 +44,25 @@ export default {
         "ignoreBot",
       ],
     },
+    iconEmoji: {
+      propDefinition: [
+        common.props.slack,
+        "icon_emoji",
+      ],
+      description: "Provide an emoji to use as filter the events. E.g. `fire`",
+      optional: false,
+    },
```

Comment:
What do you think about letting users select multiple emojis to emit events for? You could also make this prop optional, so that if the user doesn't select any emojis, it'll emit events for all emojis.
```suggestion
    iconEmoji: {
      propDefinition: [
        common.props.slack,
        "icon_emoji",
      ],
      description: "Provide an emoji to use as filter the events. E.g. `fire`",
      type: "string[]",
    },
```